server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportImport.java

```diff
@@ -207,6 +216,18 @@ public String endpoint(final Request the_request, final Response the_response) {
       return my_endpoint_result.get();
     }
     
+    // check to be sure that the same county isn't in the middle of a CVR import
+    synchronized (COUNTIES_RUNNING) {
+      if (COUNTIES_RUNNING.contains(county.id())) {
+        transactionFailure(the_response, 
```

Comment:
How is this failure handled on the client side?

Reply:
That is an excellent question. I imagine just like any other failure to perform an import.

Reply:
(it should be indistinguishable, to the client, from any other transaction failure)